backend/src/main.py

```diff
@@ -9,6 +10,15 @@
 load_dotenv()
 
 app = FastAPI()
+
+app.add_middleware(
```

Comment:
Can you explain why this was added? Sorry not too familiar with cors

Reply:
need cors middleware to access backend from frontend, but this overly loose will tighten it in the future

Reply:
Ok